Changelog.md

```diff
@@ -1,7 +1,7 @@
 # FOSSA CLI Changelog
 
-## Unreleased
-- archive: considers 0-byte tar file to be valid tar file. ([#1311](https://github.com/fossas/fossa-cli/pull/1311)) 
+## v3.8.23
```

Comment:
Actually, a couple of the older releases referenced in the changelog don't actually exist (the last actual release is v3.8.20).

Can we instead squash all these down into a single release, v3.8.21, and edit the changelog accordingly instead?

Reply:
Yes, that's a much better idea. Done!